README.md

```diff
@@ -1,13 +1,58 @@
 # Pretty Legendary SQL Client
 
-Query many database at once.
+Pretty Legendary SQL Client is graphical SQL client that allow you to query multiple database at once.
 
-# To start
+Once your connection to database are configured, the software will take care to send asked query to all configured connection and display the result.
 
- * npm install
- * npm install -g gulp
- * gulp
+That software will help you retrieves and compare quickly your data between various schema in your infrastructure.
 
-# License
+![Executing query](https://raw.githubusercontent.com/kyle-ilantzis/pl-sql-client/master/doc/pictures/query.png)
+
+## Building from source
+
+### Dependencies
+ * [Node v4.2.1](https://nodejs.org/en/)
+
+### Quick start
+
+ 1. git clone git@github.com:kyle-ilantzis/pl-sql-client.git
+ 2. npm install
+ 3. npm install -g gulp
+ 4. gulp
+
+## Download
+
+* [Releases page](https://github.com/kyle-ilantzis/pl-sql-client/releases)
+
+## Features
+
+ * Executing query on multiple database
+  * Postgres
+  * MySQL
+
+ ![Executing query](https://raw.githubusercontent.com/kyle-ilantzis/pl-sql-client/master/doc/pictures/query.png)  
+
+ * Configuring a set of database
+
+ ![Configuring database](https://raw.githubusercontent.com/kyle-ilantzis/pl-sql-client/master/doc/pictures/database1.png)
+
+![Viewing database configuration](https://raw.githubusercontent.com/kyle-ilantzis/pl-sql-client/master/doc/pictures/database2.png)
+
+ * History of executed queries
+
+![Consulting history of queries](https://raw.githubusercontent.com/kyle-ilantzis/pl-sql-client/master/doc/pictures/query.png)
```

Comment:
This is the same picture as "Executing query", we don't see the history panel expanded